common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -337,6 +337,17 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  val DiscussionAllowAnonymousRecommendsSwitch = Switch(
+    SwitchGroup.Feature,
+    "discussion-allow-anonymous-recommends-switch",
+    "if this is switched on, comments can be recommended by signed out users",
+    owners = Seq(Owner.withGithub("NathanielBennett")),
+    safeState = Off,
+    sellByDate = never,
```

Comment:
@piuccio . Yeah I was thinking this is a feature, rather than a test. If we don't want it we get rid of it.